test/integration/health.test.ts

```diff
@@ -0,0 +1,26 @@
+const chai = require("chai");
```

Comment:
Can we use ES modules instead of common js here, so that it it consistent across the repo?

Reply:
This is resolved.